osu.Game/Overlays/Comments/VotePill.cs

```diff
@@ -60,6 +58,9 @@ private void load(OsuColour colours)
         {
             AccentColour = borderContainer.BorderColour = sideNumber.Colour = colours.GreenLight;
             hoverLayer.Colour = Color4.Black.Opacity(0.5f);
+
+            if (api.LocalUser.Value.Id != comment.UserId && api.LocalUser.Value.Id != 1)
```

Comment:
What's the `!= 1` check for? I don't remember seeing that before, is local user guaranteed to always have ID 1?

Reply:
Local (logged-out) user has ID = 1

Reply:
Okay, I suppose this is fine then (there seems to be no online user with id 1 anyways).

Reply:
`api.IsLoggedIn`?